git-use-https/action.yaml

```diff
@@ -0,0 +1,12 @@
+name: "Use HTTPS for GitHub data transfer"
+description: "Use Git over HTTPS with OAuth instead of SSH to transfer data to GitHub"
+inputs:
+  token:
+    description: "OAuth 2.0 token to use for authentication"
+    required: True
+runs:
+  using: "composite"
```

Comment:
```suggestion
description: "Use Git over HTTPS with OAuth instead of SSH to transfer data to GitHub"

inputs:
  token:
    description: "OAuth 2.0 token to use for authentication"
    required: True

runs:
  using: "composite"
```